### DISCUSSION TITLE
Help - Unable to view webpage from browser.

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1689] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Just chatting

### Description

Hello..... 

When I head to my domain after a push, i receive this 

"--- layout: home # Index page --- " dominicspucches.com 

Im sure its simple but my knowledge with this area is pretty null.

-Dom

**Comment by huanyushi** [comment]
See [here](https://chirpy.cotes.page/posts/getting-started/#deploy-by-using-github-actions).

This question has been repeated many times. If that doesn't work for you, maybe you need to describe exactly what you did.
